### PR TITLE
github and deploy commands

### PR DESCRIPTION
**Pull request by dbaltas**

### CODE REVIEW

File: github/github.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"context"
+	"fmt"
+
+	"github.com/dbaltas/ergo/repo"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+//CreateDraftRelease creates a draft release
+func CreateDraftRelease(ctx context.Context, accessToken string, organization string,
```
Comment (mantzas) [inline github/github.go:13]: you should create a struct to contain all arguments

File: main.go [hunk 2]
```diff
@@ -64,11 +119,39 @@ func main() {
...
 	}
-	printBranchCompare(diff)
+
+	if deployCmd {
+		deployBranches(baseBranch, branches, releaseOffset, releaseInterval, directory)
+		return
+	}
+
+	if createReleaseCmd {
+		tagName := "2018.04.22"
```
Comment (mantzas) [inline main.go]: hardcoded?

File: main.go [hunk 3]
```diff
@@ -64,11 +119,39 @@ func main() {
 		diff = append(diff, branchCommitDiff)
 	}
 
-	if inDetail {
+	if statusCmd {
```
Comment (mantzas) [inline main.go:160]: you should use cobra for commands, subcommands and flags

File: main.go (hunk 2 above)
Comment (dbaltas) [reply main.go]: 👍

File: github/github.go (hunk 1 above)
Comment (dbaltas) [reply github/github.go:13]: Will do so after switching to commands.

**Pull request merged**